canonical-websites/tutorials.ubuntu.com
# Fix link on custom lxd

## Done

- Fixed link to another tutorial

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/tutorial/create-custom-lxd-images#1](http://0.0.0.0:8016/tutorial/create-custom-lxd-images#1)
- See that the link goes to https://tutorials.ubuntu.com/tutorial/tutorial-setting-up-lxd-1604#0

## Issue / Card

Fixes #842
